codelists/open/impact_unit.csv

```diff
@@ -0,0 +1,11 @@
+Code,Title
+percentage,Percentage
+hour,Hour
+day,Day
+week,Week
+month,Month
+year,Year
+count,Count
+m2,Meters squared
```

Comment:
I'm not changing this as a square meter is different from meters squared, e.g. 2 meters squared = 4 square meters

Reply:
That's true, but the [SI-derived unit of area is a square meter](https://simple.wikipedia.org/wiki/Square_metre) and that is what the symbol m² represents. I'm assuming that is what was meant by m2 in the codelist since, to my knowledge, area is not commonly measured in meters squared, but it is commonly measured in square meters. I'll defer to @stufraser1 in case it is common practice to measure area in meters squared in risk datasets.

Reply:
fair enough :)